LEGO1/mxentity.cpp

```diff
@@ -1,6 +1,6 @@
 #include "mxentity.h"
 
-DECOMP_SIZE_ASSERT(MxEntity, 0x68)
+// DECOMP_SIZE_ASSERT(MxEntity, 0x68)
```

Comment:
I think we should leave this in place and just adjust it to specify the currently assumed size (with a comment that it might still change in the future), so we can still prevent any silent unintended changes to the size of `MxEntity` in the future.